src/modules/IscsiClientLib.rb

```diff
@@ -553,7 +554,7 @@ def start_iscsid_initial
           i,
           Ops.get_integer(cmd, "exit", -1)
         )
-        if Ops.get_integer(cmd, "exit", -1) == 0
+        if Ops.get_integer(cmd, "exit", -1) == 0 || Ops.get_integer(cmd, "exit", -1) == 21
```

Comment:
Please, please, add a comment explaining why the hell 21 is an accepted result mentioning bsc#1131049. Otherwise, over time that 21 will become just another magic number thrown in the code nobody knows why. 

On the other hand, I believe this would be much more readable as

```ruby
if [0, 21].include?(cmd["exit"])
```

Or maybe

```ruby
if cmd["exit"].zero? || cmd["exit"] == 21
```

Or, going even further, creating a separate method to make this look like

```ruby
if iscsid_success?(cmd)
```

Reply:
I explained it in the commit message. I thought that'd be sufficient.

Wrt the ruby syntax, by all means, change it! I'm a total noob when in comes to ruby.